crates/sui-json-rpc-types/src/sui_object.rs

```diff
@@ -447,6 +447,7 @@ impl TryInto<Object> for SuiObjectData {
                 self.version,
                 &p.module_map,
                 protocol_config.max_move_package_size(),
+                &[],
```

Comment:
In this case, I would expect `SuiRawData::Package` to include the type origin table somewhere within it -- i.e. this should be a straight type conversion.